ckanext/blob_storage/storage.py

```diff
@@ -0,0 +1,9 @@
+class ResourceBlobStorage(object):
```

Comment:
Let's rename this class to `DummyUploader` or `StubUploader` and module to `uploader.py` from `storage.py`.
Also a piece of comment explaining why this class is a stub would be really helpful for the future. E.g.
```
class StubUploader:
  # stub implementation to overwrite default CKAN uploader in order to inform ckanext-validation that it's cloud storage implemenation see: ###
```

Reply:
Done, although I've decided to go with a comment that's more Fjelltopp agnostic :)